Documentation/Signoffs/SteeringSystem.md

```diff
@@ -0,0 +1,107 @@
+STEERING WHEEL SUBSYSTEM
+------------------------
+### Function of the Subsystem
+The Steering Wheel subsytem shall recieve the input of the Nissan Leaf steering wheel into a sensor
+embedded into the steering column of the vehicle. The sensor will measure the steering angle and send
+the data into a local microcontroller which will interpret the data and relay it to the simulation.
+
+### Specifications & Constraints
```

Comment:
I think there are further constraints/specs that are appropriate. Like not loose position when power cycled?